pkg/executor/executortype/poolmgr/gpm.go

```diff
@@ -566,94 +568,94 @@ func (gpm *GenericPoolManager) getFunctionEnv(ctx context.Context, fn *fv1.Funct
 
 // idleObjectReaper reaps objects after certain idle time
 func (gpm *GenericPoolManager) idleObjectReaper() {
```

Comment:
Please pass context to this function from calling function.

Reply:
done